docker-compose-files/dspace-oracle-compose/README.md

```diff
@@ -125,11 +125,6 @@ _When DSpace is restarted, the contents of your volumes will be restored_
 docker-compose -p $DPROJ up -d
```

Comment:
Running this command showed the `docker-compose` help output. It is possible I missed a step somewhere. Also, there seems to be looks of changes that remove `$DPROJ` environment variable, would it be possible to do the same here?

Reply:
@PTrottier , I removed the the DPROJ variable from most of the code files.  Since this Oracle module is experimental, I did not apply the same refactoring here.  I would be inclined to create a new ticket to refactor this file once the main PR has been accepted and merged.